### DISCUSSION TITLE
Contribute

### DISCUSSION THREAD

**Discussion by carla-fiadeiro** [Azure/mlops-v2#7] [Ideas]
Hi, I would like to contribute but I would like to know what exactly do you need help with.

Thank you,
Carla

**Comment by setuc** [comment]
Hi @carla-fiadeiro I completely missed the discussion topic. We will be creating the plan on GitHub and would welcome the contributions. Some of the potential areas could be around

* GitHub actions
*  Security / SecDevOps
*  Azure Security
*  Azure DevOps
